docs/modules/ROOT/pages/index.adoc

```diff
@@ -287,6 +287,12 @@ Hard bindings are described in link:++https://c2pa.org/specifications/specificat
 
 The _<<_actor,actor>>_ that has control (specifically signature authority) over a _<<ToIP verifiable identifier>>_ that describes a specific _<<_named_actor,named actor>>._
 
+==== Credential revocation
+
+An action signifying that a digital credential can no longer be considered as valid. This is typically indicated by an action on the part of the credential issuer stating that they can no longer attest to the validity of the information in the credential or an action on the part of the _<<_credential_holder,credential holder>>_ that the public/private key pair associated with the credential is no longer valid.
```

Comment:
Something is missing before "that the public/private key". E.g., "The result is that the public/private key"

Reply:
UPDATE to say … "_stating_ that the public/private key pair ..."